src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx

```diff
@@ -11,6 +11,7 @@ import translate from '../../../../../utils/translations/Translations';
 class ArtistPage extends React.Component {
   constructor(props) {
     super(props);
+    this.ref = React.createRef();
```

Comment:
Why adding a ref ?

Reply:
Inside `render()` ( https://github.com/Musish/Musish/blob/fb582fe5a52e8a6b5b74ad10f6caaa0c13e2433a/src/app/components/Inside/Pages/Catalog/Artist/ArtistPage.jsx#L125 ) it is used but was never defined.

Reply:
The ref is unused, can you remove it from the constructor and the `PageContent` ?

Reply:
@raphaelvigee if I remove it I get an error in console.log:

```
checkPropTypes.js:20 Warning: Failed prop type: The prop `innerRef` is marked as required in `PageContent`, but its value is `undefined`.
    in PageContent (created by ArtistAlbums)
    in ArtistAlbums (created by Route)
    in Route (created by ArtistsPage)
    in ArtistsPage (created by Route)
    in Route (created by App)
    in Switch (created by App)
    in main (created by Layout)
    in div (created by Layout)
    in Layout (created by Context.Consumer)
    in ContextComponent (created by Route)
    in Route (created by withRouter(ContextComponent))
    in withRouter(ContextComponent) (created by App)
    in MusicKitAuthorizeProvider (created by Enhance)
    in Enhance (created by App)
    in GoogleAnalyticsProvider (created by Route)
    in Route (created by withRouter(GoogleAnalyticsProvider))
    in withRouter(GoogleAnalyticsProvider) (created by App)
    in Router (created by BrowserRouter)
    in BrowserRouter (created by App)
    in MusicKitProvider (created by App)
    in App (created by DragDropContext(App))
    in DragDropContext(App)
```

Reply:
@raphaelvigee I have removed the usage of innerRef and it breaks the app. This prop was introduced in https://github.com/Musish/Musish/commit/39f7fb89817436a0a0077a3d53cd5724a1b5c533 in order to fix the scroll.

Without it I get an error in console:
```Uncaught TypeError: Cannot read property 'scrollHeight' of null
    at InfiniteLoader.componentDidUpdate (InfiniteLoader.jsx:95)```

I guess we cannot remove it.

Reply:
@raphaelvigee so how should we proceed?

Reply:
@raphaelvigee any ideas?

Reply:
@SpartakusMd Sorry for not getting back to you on this, I believe this has been fixed in the latest TS refactoring, if not, feel free to open a new PR